.github/workflows/release.yml

```diff
@@ -261,28 +265,32 @@ jobs:
   upload_golem_windows:
       name: 🚀 [Windows] Golem
       needs: create_release # Depends on release being created for upload URL
-      runs-on: windows-latest
+      runs-on: ubuntu-latest
       steps:
-        - name: 👾 Disable defender
-          shell: powershell
-          run: start-process -filepath powershell -ArgumentList '/c','Set-MpPreference -DisableRealtimeMonitoring $true' -verb RunAs
         - uses: actions/checkout@v3
         - name: 🛠️ Setup Rust Toolchain
           uses: actions-rs/toolchain@v1
           with:
-            toolchain: nightly-2023-09-04
+            toolchain: '1.76.0'
             default: true
             profile: minimal
-        - name: ⚡ Setup Rust (Loader)
-          shell: powershell
+        - name: 🛠️ Setup Rust Toolchain (Loader)
+          uses: actions-rs/toolchain@v1
+          with:
+            toolchain: nightly-2024-02-11
```

Comment:
Should this be 1.76.0 instead os still being on nightly?

Reply:
Naw loader is special :(